src/component/base/DraftEditor.react.js

```diff
@@ -249,8 +255,8 @@ class DraftEditor extends React.Component<DraftEditorProps, State> {
       });
     }
 
-    // See `restoreEditorDOM()`.
-    this.state = {contentsKey: 0};
+    // See `restoreEditorDOM()` and `restoreComposedBlocksDOM()`
+    this.state = {contentsKey: 0, blocksKey: 0};
```

Comment:
@bchmn Since this is only a number that will be added to a block key, and not an actual block/blocks key, I am wondering about the name. Maybe blocksKeyCounter?